composer.json

```diff
@@ -6,21 +6,21 @@
         "smarty/smarty" : "~3.1.33",
         "firebase/php-jwt" : "~3.0",
         "google/recaptcha": "~1.1",
-        "php-http/guzzle6-adapter": ">=2.0.0",
+        "php-http/guzzle7-adapter": "0.1.1",
         "psr/http-message": "~1.0",
         "psr/http-server-handler" : "*",
         "psr/http-server-middleware" : "*",
-        "laminas/laminas-diactoros" : "~1.8.4",
+        "laminas/laminas-diactoros" : "2.5.0",
         "ext-json": "*",
-        "bjeavons/zxcvbn-php": "^0.4"
+        "bjeavons/zxcvbn-php": "^1.0"
     },
     "require-dev" : {
-        "squizlabs/php_codesniffer" : "3.5.3",
-        "phpunit/phpunit" : "7.0.0",
-        "phan/phan": "3.0.0",
+        "squizlabs/php_codesniffer" : "3.5.8",
+        "phpunit/phpunit" : "9.4.4",
+        "phan/phan": "4.0.0-alpha5",
```

Comment:
Notice phan/phan is using 4.0.0-alpha5. 3.0.0 doesn't support php 8 and I believe the plan for php support by phan is for the future 4.0.0 release.

Reply:
How are you getting past phpmd? Are you just manually running phan? When I run `make checkstatic` on this branch it dies on phpmd before getting to phan.